tests/_core/test_plot.py

```diff
@@ -1852,6 +1852,12 @@ def test_1d_column_wrapped(self):
         for s in subplots[1:]:
             ax = s["ax"]
             assert ax.xaxis.get_label().get_visible()
+            # mpl3.7 added a getter for tick params, but both yaxis and xaxis return
+            # the same entry of "labelleft" instead of  "labelbottom" for xaxis
+            if not _version_predates(mpl, "3.7"):
+                assert ax.xaxis.get_tick_params()["labelleft"]
+            else:
+                assert len(ax.get_xticklabels()) > 0
```

Comment:
I'm confused about how this is flagged as not tested since the pinned build log appears to have matplotlib 3.4 installed.

Reply:
Unsure about this comment by codecov, but its webpage for this test suite suggests it is tested (lines marked as green). See [here](https://app.codecov.io/gh/mwaskom/seaborn/pull/3600/blob/tests/_core/test_plot.py#L1860).

Reply:
Agreed, the full code coverage report looks fine, not sure what's up with the bot.